services/user.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = "",
+        last_name: Optional[str] = ""
+) -> User:
+    user_model = get_user_model()
+    user = user_model.objects.create_user(
+        username=username,
+        password=password,
+        email=email,
+        first_name=first_name,
+        last_name=last_name
+    )
+
+    return user
+
+
+def get_user(user_id: int) -> Optional[User]:
+    try:
+        return get_user_model().objects.get(pk=user_id)
+    except User.DoesNotExist:
+        return None
```

Comment:
So this is bad design - imagine you call your function on line 42 and user does not exist. The program will continue working and try to access `.username` or other fields on `None` object which will lead to errors.
Be careful with exceptions - when to use them and when not to, and also how to correctly process them.